.editorconfig

```diff
@@ -1,11 +1,5 @@
 root = true
 
-[*]
-end_of_line=lf
-charset=utf-8
-trim_trailing_whitespace=true
-insert_final_newline=true
-
```

Comment:
This looks like a broad stroke. If the problem was with binary files, maybe it is the utf-8 part?

If it isn't that easy, I'm okay removing the `[*]` table, but we should at least add these rules to each of the explicitly mentioned formats below.

Reply:
I agree that we shouldn't have to remove this. The binary file being a problem is actually ec-linux-amd64 itself and the error is about trailing whitespaces see (https://github.com/PureStake/moonbeam/runs/1717959278?check_suite_focus=true)

All the other types are explicitly specified so I don't worry too much about it, but there could be a better solution